src/index.js

```diff
@@ -25,15 +24,15 @@ class AudioPlayer {
     constructor (audioEngine) {
         this.audioEngine = audioEngine;
 
-        // effects setup
+        // Create the audio effects
         this.pitchEffect = new PitchEffect();
-        this.panEffect = new PanEffect();
+        this.panEffect = new PanEffect(this.audioEngine.context);
 
-        // the effects are chained to an effects node for this player, then to the main audio engine
-        // audio is sent from each soundplayer, through the effects in order, then to the global effects
-        // note that the pitch effect works differently - it sets the playback rate for each soundplayer
-        this.effectsNode = new Tone.Gain();
-        this.effectsNode.chain(this.panEffect, this.audioEngine.input);
+        // Chain the audio effects together
+        // effectsNode -> panEffect -> audioEngine.input -> destination (speakers)
```

Comment:
This comment may be slightly misleading because there is actually no connection to the destination in the code (I'm guessing that is done by the audio engine elsewhere?).